superset/charts/post_processing.py

```diff
@@ -230,8 +230,6 @@ def pivot_table_v2(
     Pivot table v2.
     """
     verbose_map = datasource.data["verbose_map"] if datasource else None
-    if form_data.get("granularity_sqla") == "all" and DTTM_ALIAS in df:
```

Comment:
This is likely a bug (see like #259), i.e., it likely should have been `granularity` rather than `granularity_sqla` as this represents the Druid NoSQL time grain (as opposed to the SQL time column) where `all` is a Druid NoSQL concept which represents a single time grain/bucket to encapsulate the entire time range.